lib/check-version.js

```diff
@@ -36,3 +36,17 @@ module.exports = function (done) {
     done()
   })
 }
+
+function warnForVue2Version (template, name) {
+  var vue1InitCommand = 'vue init ' + template + '#1.0' + ' ' + name
```

Comment:
Should we also pass in `inPlace` and then only show the `name` in the command if it's false?